src/utils/history.py

```
@@ -201,167 +169,196 @@ async def create_summary(
-
-async def get_full_history(
-    db: AsyncSession, session_id: str
-) -> tuple[str, list[models.Message]]:
-    """
-    Get all messages for a given session.
-    """
-    messages = await get_messages_since_message(db, session_id)
-    return format_messages(messages), messages
 
 
 async def get_summarized_history(
```

Pylint (3.3.7)

[refactor] Too many arguments (6/5)

(R0913)